raiden/connection_manager.py

```diff
@@ -126,34 +143,48 @@ def leave(self, wait_for_settle=True, timeout=30):
                 c.partner_address) for c in open_channels]
             for channel in channel_specs:
                 try:
-                    self.api.close(*channel),
-                except RuntimeError:
+                    self.api.close(*channel)
+                except:
```

Comment:
I also wanted to comment on this but then saw that it's reraised so I think it can be okay in that case?